cxx/structureTranslationModule/agent/StructureTranslationAgent.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "sc-memory/kpm/sc_agent.hpp"
+
+/*
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
```

Comment:
includes and pragma statement should be after licence header comment, Somewhere you forgot to add licence (in cpp or hpp files). Check everywhere